website/src/routes/api/(schemas)/string/index.mdx

```diff
@@ -2,6 +2,116 @@
 title: string
 ---
 
+import { Property } from '~/components';
+import { Link } from '@builder.io/qwik-city';
+
 # string
 
-> The content of this page is not yet ready. Until then just use the [source code](https://github.com/fabian-hiller/valibot/blob/main/library/src/schemas/string/string.ts).
+Schema that parses the input to string.
+
+```ts
+const schema = string(errorMessage, pipe);
```

Comment:
Here it is important to me that we represent each overload signature and use the same parameter names as in the code:

> I will probably rename the `error` parameter to `message` in the next days.

```ts
// String schema with pipe
const schema = string(pipe);

// String schema with message and pipe
const schema = string(message, pipe);
```